design/capm3-remediation-controller-proposal.md

```diff
@@ -104,24 +104,28 @@ Metal3RemediationTemplate.
 #### Remediation Controller
 
 * MRC watches for the presence of Metal3Remediation CR.
-* Based on the remediation strategy defined in ```.spec.strategy``` in
+* Based on the remediation strategy defined in ```.spec.strategy.type``` in
   Metal3Remediation, RC uses BMO APIs to get hosts back into a healthy or
   manageable state.
 * RC uses ```.status.phase``` to save the states of the remediation. Available
-  states are running, rebooting, power off, failed.
+  states are running, waiting, deleting machine.
```

Comment:
what does `waiting` mean? waiting for what?
I'm also not sure about `running`. maybe it worth to describe each state?

Reply:
Perhaps `in-progress` or `active` instead of `running`
IIRC `waiting` meant that we'd performed an action (like reboot) and we're waiting to see if the node returns before the timeout.

Reply:
Yes, `waiting` is state where we have reboot the host and we wait the timeout. `Running` is the state where our controller has detected the remediation request but the first reboot is not done yet.